core/src/main/java/inetsoft/sree/PropertiesEngine.java

```diff
@@ -1021,6 +1031,11 @@ public void entryRemoved(KeyValueStorage.Event<String> event) {
 
       private void onChange(KeyValueStorage.Event<String> e) {
          PropertyChange change = new PropertyChange(e.getKey(), e.getOldValue(), e.getNewValue());
+
+         if(Tool.equals(e.getKey(),"format.number.round")){
```

Comment:
There's no need to constrain this to a particular property. Fire the property change event for all properties.